### PR TITLE
Setup SAAT and SIDAM

### PR DESCRIPTION
**Pull request by tchow8**
[DIV-4139](https://tools.hmcts.net/jira/browse/DIV-4319)

Mainly Integration Test Refactor to Support SIDAM

### CODE REVIEW

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java [hunk 1]
```diff
@@ -43,7 +44,7 @@ private void createUserInIdam(String username, String password) {
             .forename("Test")
             .surname("User")
             .roles(new UserCode[] { UserCode.builder().code("citizen").build() })
-            .userGroup(UserCode.builder().code("divorce-private-beta").build())
+            .userGroup(UserCode.builder().code("citizens").build())
             .build();
 
         RestAssured.given()
```
Comment (qzhou-hmcts) [inline src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java]: SerenityRest

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java [hunk 2]
```diff
@@ -2,12 +2,13 @@
 
 import io.restassured.RestAssured;
```
Comment (qzhou-hmcts) [inline src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java]: Remove this import

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java (hunk 1 above)
Comment (tchow8) [reply src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java]: done

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java (hunk 2 above)
Comment (tchow8) [reply src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java]: done

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java [hunk 3]
```diff
@@ -43,18 +43,18 @@ private void createUserInIdam(String username, String password) {
...
+        SerenityRest.given()
             .header("Content-Type", "application/json")
             .body(ResourceLoader.objectToJson(userRequest))
             .post(idamCreateUrl());
     }
 
     private void createUserInIdam() {
         idamUsername = "simulate-delivered" + UUID.randomUUID() + "@notifications.service.gov.uk";
-        idamPassword = UUID.randomUUID().toString();
+        idamPassword = "genericPassword123";
```
Comment (qzhou-hmcts) [inline src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java]: use a constant

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/IntegrationTest.java [hunk 4]
```diff
@@ -67,7 +67,7 @@ String getDocumentStoreURI(String uri) {
 
     private synchronized String getUserToken() {
         username = "simulate-delivered" + UUID.randomUUID() + "@notifications.service.gov.uk";
-        String password = UUID.randomUUID().toString();
+        String password = "genericPassword123";
```
Comment (qzhou-hmcts) [inline src/integrationTest/java/uk/gov/hmcts/reform/divorce/IntegrationTest.java]: use a constant

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java (hunk 3 above)
Comment (tchow8) [reply src/integrationTest/java/uk/gov/hmcts/reform/divorce/IdamUtils.java]: Done

File: src/integrationTest/java/uk/gov/hmcts/reform/divorce/IntegrationTest.java (hunk 4 above)
Comment (tchow8) [reply src/integrationTest/java/uk/gov/hmcts/reform/divorce/IntegrationTest.java]: Done

**Pull request merged**